index.html

```diff
@@ -6,17 +6,51 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-    <form>
-      <input type="text" id="slug" />
-      <input type="submit" value="Slug it" onclick="slugIt(); return false;"></input>
+    <form id="form" onSubmit="" style="max-width:60ch">
+			<fieldset>
+
+				<label>Input text</label>
+      <input type="text" id="input" />
+			</fieldset>
+
+
+
+			<fieldset>
+
+				<label for="replacement">Replacement char</label>
+				<input type="text" id="replacement" name="replacement" maxlength="1" />
```

Comment:
Does it make sense to put the default value as placeholder text for this and other configuration fields?